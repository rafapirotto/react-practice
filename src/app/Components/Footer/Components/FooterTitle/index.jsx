import React from 'react';
import PropTypes from 'prop-types';

const propTypes = {
  title: PropTypes.string.isRequired,
};

const FooterTitle = ({ title }) => (
  <div
    className="
              flex
              pt-24
              text-4xl
              sm:justify-center
              sm:pl-0
              md:justify-center
              md:pl-0
              lg:justify-start
              lg:pl-40
          "
  >
    {title}
  </div>
);

FooterTitle.propTypes = propTypes;

export default FooterTitle;
