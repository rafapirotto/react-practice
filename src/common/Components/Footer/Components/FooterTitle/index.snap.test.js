import React from 'react';
import renderer from 'react-test-renderer';

import FooterTitle from '.';

it('matches the snapshot', () => {
  const component = renderer.create(<FooterTitle title="example" />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
