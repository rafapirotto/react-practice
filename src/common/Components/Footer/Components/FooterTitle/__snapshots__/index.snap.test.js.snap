// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<div
  className=" flex pt-24 text-4xl sm:justify-center sm:pl-0 md:justify-center md:pl-0 lg:justify-start lg:pl-40 "
>
  example
</div>
`;
